alibi_detect/saving/schemas.py

```diff
@@ -87,7 +91,7 @@ class DetectorConfig(CustomBaseModel):
     "Name of the detector e.g. `MMDDrift`."
     backend: Literal['tensorflow', 'pytorch', 'sklearn'] = 'tensorflow'
     "The detector backend."
-    meta: Optional[MetaData] = MetaData()
+    meta: Optional[MetaData]
```

Comment:
Why change from setting to `MetaData()` ?

Reply:
The original was sort of a hack to get `pydantic` to set the `self.config['meta']` field. This is now set in its entirety by `_set_config` and `get_config`. IMO `meta: Optional[MetaData]` is more natural, and this is how we do all the other pydantic nested models like `ModelConfig` etc.